oci-seccomp-bpf-hook.go

```diff
@@ -190,7 +205,7 @@ func startFloatingProcess() error {
 			return fmt.Errorf("cannot determine executable path:%q", err.Error())
 		}
 
-		process, err := os.StartProcess(executable, []string{"oci-seccomp-bpf-hook", "-r", strconv.Itoa(pid), "-f", fileName}, attr)
+		process, err := os.StartProcess(executable, []string{"oci-seccomp-bpf-hook", "-r", strconv.Itoa(pid), "-o", outputFile, "-i", inputFile}, attr)
```

Comment:
It shouldn't matter much, but for completeness: `s/"oci-seccomp-bpf-hook"/executable/`